kornia/augmentation/augmentation.py

```diff
@@ -387,10 +388,10 @@ def __init__(
             warnings.warn("interpolation is deprecated. Please use resample instead.", category=DeprecationWarning)
             self.resample = Resample.get(interpolation)
         self.resample = Resample.get(resample)
-        self.align_corners = align_corners
+        self.align_corners = _parse_align_corners(align_corners, self.resample, old_default=False)
```

Comment:
add typing

Reply:
Not sure what you mean by that. Could you elaborate?

Reply:
`self.align_corners: Optional[bool]` because I know what `_parse_align_corners` does and what return

Reply:
Why do you want to have this? `mypy` doesn't need it, since it can infer the type from the return type of `_parse_align_corners`.

Reply:
readability